main.slide

```diff
@@ -154,7 +156,7 @@ Le noeud sur lequel est déployé notre pod est quasiment saturé en mémoire. D
 kubectl apply -f src/conso-256m.yaml
 ```
 
-On trouve des replicas sur le noeud saturé! Et un des pods a redémarré pour cause d'OOMKill.
+On trouve des replicas sur le noeud saturé! Et un des pods a redémarré pour cause d'OOMKill (définition orale ptet suffisante).
```

Comment:
Ou le mettre en toutes lettres en fait, Out of Memory kill ca parlera surement plus